src/ngp_algorithms/WallDistGradElemAlg.C

```diff
@@ -0,0 +1,141 @@
+// Copyright 2017 National Technology & Engineering Solutions of Sandia, LLC
+// (NTESS), National Renewable Energy Laboratory, University of Texas Austin,
+// Northwest Research Associates. Under the terms of Contract DE-NA0003525
+// with NTESS, the U.S. Government retains certain rights in this software.
+//
+// This software is released under the BSD 3-clause license. See LICENSE file
+// for more details.
+//
+
+
+#include "ngp_algorithms/WallDistGradElemAlg.h"
```

Comment:
Question for those who know better... do we want a `GradElem`? Didn't we stop developing the elem stuff?

Reply:
I'm a little torn about this.  I'm inclined to leave it for now since @wilsonrvornlgov has already put it in.  We still have a lot of the elem scheme infrastructure, and we use some of it for PNG and other things.  I say leave it in as part of this PR so it makes it into the git history.  We can delete it later if we are never using it.

Reply:
Thanks for the feedback. I let this as is.